Kerpecheva/remove token command (#947)
telerik/maui-docs: controls/combobox/commands.md

# .NET MAUI ComboBox Commands

The [.NET MAUI ComboBox]({%slug combobox-overview%}) provides a few commands that will help you manipulate its selection.

## SelectAll Command

- `SelectAllCommand` (`ICommand`)&mdash;Selects all items from the source.

> > The `SelectAll` command can be used only when `SelectionMode` is `Multiple`. In case the command is invoked in `Single` `SelectionMode`, the ComboBox throws an exception.

Reviewer: - Avoid the passive voice.
- Avoid the visual clutter caused by the close placement of different styles (headings, inline code, a note)&mdash;as the section contains only a single line of text (except the content of the note), you don't need to use the note style to grab the attention of the reader. 
```suggestion
To use the `SelectAll` command, set the selection mode to `Multiple`. If you invoke the command in `Single` selection mode, the ComboBox throws an exception.
```

## Clear Selection Command

> - `ClearSelectionCommand` (`ICommand`)&mdash;Sets the selection to null. If Multiple `SelectionMode` is used, this command will clear all selected items. YOu can override the default behavior and create custom command.

Reviewer: ```suggestion
`ClearSelectionCommand` (`ICommand`)&mdash;Sets the selection to null. If you use the `Multiple` selection mode, this command will clear all selected items. You can override the default behavior and create a custom command.
```

![ComboBox ClearSelectionCommand](images/combobox-commands.gif)

> ## Remove Token Command

Reviewer: The suggestions from the `controls/autocomplete/commands.md` article apply here too.
